Editor/Gui/UiSymbolJson.cs

```diff
@@ -19,32 +19,34 @@
 
 namespace T3.Editor.Gui
 {
-    public class SymbolUiJson 
+    public static class SymbolUiJson
     {
-        public JsonTextWriter Writer { get; set; }
-        public JsonTextReader Reader { get; init; }
+        private static readonly Func<JToken, object> JsonToVector2 = JsonToTypeValueConverters.Entries[typeof(Vector2)];
+        private static readonly Func<JToken, object> JsonToVector4 = JsonToTypeValueConverters.Entries[typeof(Vector4)];
+        private static readonly Action<JsonTextWriter, object> Vector2ToJson = TypeValueToJsonConverters.Entries[typeof(Vector2)];
+        private static readonly Action<JsonTextWriter, object> Vector4ToJson = TypeValueToJsonConverters.Entries[typeof(Vector4)];

```

Comment:
I'm kind of confused. Wouldn't those properties be initialized right at be beginning when `JsonToTypeValueConverters` might not event be initialized in `Model.cs`?  How does this work 🤔  ?

Reply:
I'm pretty sure I'm relying on the runtime initialization order here. Since this class is never referenced before `UiModel`'s static constructor would be, these variables are guaranteed to have their intended values. My totally uninformed guess is that static values like this are initialized in the same way as a static constructor, so they wouldn't be evaluated until the `UiModel` references the `UiSymbolJson` class anyway